apps/workflow-helper.sh

```diff
@@ -71,8 +74,14 @@ docker run --privileged -t --name "tink-worker" \
 	-e "TINKERBELL_CERT_URL=$grpc_cert_url" \
 	-e "REGISTRY_USERNAME=$registry_username" \
 	-e "REGISTRY_PASSWORD=$registry_password" \
+	-e "LOG_DRIVER=$log_driver" \
+	-e "LOG_OPT_SERVER_ADDRESS=$log_opt_server_address" \
+	-e "LOG_OPT_TAG=$log_opt_tag" \
 	-v /worker:/worker \
 	-v /var/run/docker.sock:/var/run/docker.sock \
 	-t \
 	--net host \
+	--log-driver "$log_driver" \
+	--log-opt "$log_driver"-address="$log_opt_server_address" \
```

Comment:
last nit: Please just do one set of `"` for the whole arg.

Reply:
done